include/gunrock/algorithms/bc.hxx

```diff
@@ -177,10 +197,15 @@ struct enactor_t : gunrock::enactor_t<problem_t> {
         operators::advance::execute<operators::load_balance_t::merge_path,
                                     operators::advance_direction_t::forward,
                                     operators::advance_io_type_t::vertices,
-                                    operators::advance_io_type_t::none>(
+                                    operators::advance_io_type_t::vertices>(
```

Comment:
This is a significant change in terms of the algorithm and should not be required in my opinion, Let's discuss this in detail on slack.

Reply:
Also see my comment in `bfs`.